applications/ContactStructuralMechanicsApplication/custom_external_libraries/bprinter/table_printer.h

```diff
@@ -0,0 +1,213 @@
+#ifndef BPRINTER_TABLE_PRINTER_H_
+#define BPRINTER_TABLE_PRINTER_H_
+
+#include <iostream>
+#include <iomanip>
+#include <vector>
+#include <string>
+#include <sstream>
+#include <cmath>
+#if defined(USE_BOOST_KARMA)
```

Comment:
please do not include other boost libraries, particularly not spirit

Reply:
I will remove it in a future PR, rightly now are not used anyway